### ISSUE TITLE
Twig-only theme not working

### ISSUE THREAD

**Issue by markusu49** [simplesamlphp/simplesamlphp#1352]
**Describe the bug**
We're on 1.16.2 and have a working twig-only theme which consists of a single file `/modules/mytheme/themes/mytheme/core/loginuserpass.twig`. After upgrading SimpleSAMLphp, the default theme is displayed instead.

**To Reproduce**
Create the theme as described above and enable it.

**Expected behavior**
The theme should still show as before.

**Screenshots or logs**
```
/module.php/core/loginuserpass.php - Template: Could not find template file [core:loginuserpass.tpl.php] at [/modules/mytheme/themes/mytheme/core/loginuserpass] - now trying the base template
```

**Additional context**
We're upgrading from 1.16.2, the issue exists at least since 1.17.7 (earlier not tried).

**Comment by tvdijen** [comment]
Hi @markusu49 !  Twig was only supported experimentally in 1.16, so our implementation in 1.17 has become a bit different from what it was initially..
You should be able to fix your issue by adding `'usenewui' => true` to the config.php file.
The option doesn't exist in the config, so you have to add it. Let me know if this fixes things for you!

**Comment by markusu49** [comment]
Hi @tvdijen ,

yes, that seems to work. Should probably be mentioned in the [docs](https://simplesamlphp.org/docs/development/simplesamlphp-theming)

**Comment by tvdijen** [comment]
We have a bunch of new functionality that is scheduled to be officially released in 1.19, providing an upgrade path to 2.0 where all the old stuff will be removed.. I think we hadn't anticipated on people using this in 1.16 already 🤣 
I'm glad everything works out for you now! Thanks!
